docs/INSTALLATION.md

```diff
@@ -18,7 +18,7 @@ You may need to prefix `sudo` to the start of the command to install to a locati
 
 ~~~~exercism/caution
 Racket may also be available through your distribution's package manager, although it may be older than the latest Racket version.
-While you may choose to use your package manager's version now and upgrade to the current version later, as a learner you will have a much better experience if you stick to the current release available from [the official download site][racket-dl].
+While you may choose to use your package manager's version now and upgrade to the current version later, as a learner you will have a much better experience if you stick to the current release available from the official download site.
```

Comment:
```suggestion
While you may choose to use your package manager's version now and upgrade to the current version later, as a learner you will have a much better experience if you stick to the current release available from [the official download site][racket-dl].

[racket-dl]: https://download.racket-lang.org
```

(Don't forget to delete the other `[...]: ...`.)

Reply:
That was my original intention because Glenn mentioned on Discord that this section is parsed separately from the rest of the document. However. I couldn't get it working locally in a Markdown preview so I didn't want to push that change in the off-chance it works. If you're sure it works, I'll accept the change though.

Reply:
I have encountered this same problem several times, and each time this fix was succesful, e.g. [here](https://github.com/exercism/haskell/pull/1146/files).

The empty line between the text and the reference is essential.